pkg/sysctl/sysctl.go

```diff
@@ -93,6 +93,15 @@ func writeSysctl(name string, value string) error {
 	if err != nil {
 		return err
 	}
+	// Check if the value is already set to the desired value.
+	val, err := os.ReadFile(path)
+	if err != nil {
+		return fmt.Errorf("failed to read %s: %s", path, err)
```

Comment:
Could you please change the error message to the same form as other error messages in this function, smth like
```
fmt.Errorf("could not read the sysctl file %s: %s", path, err)
```

Reply:
Done.